<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoundDeviceDialog</class>
 <widget class="QDialog" name="FoundDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(230, 228, 232);
	font-size:18px;
	color: rgb(70, 79, 97);
	font-family:Microsoft Yahei;
    text-align:center;
}


QLabel
{
  	font-size:18px;
	color: rgb(255, 255, 255);
	border-color: rgba(255, 255, 255, 0);
	background-color: rgba(255, 255, 255, 0);
	font-family:Microsoft Yahei;
}
QPushButton
{
	background-color: rgba(155, 167, 179, 0);
	font:bold;
	border-color: rgb(176, 186, 221);
	border-width: 2px;
    border-style: solid;
}
QPushButton:hover
{
   border-color: rgb(255, 255, 255);
}
QPushButton:pressed
{
   border-color: rgb(255, 255, 255);
}




QScrollBar:vertical
{
    width:17px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:17px;
    padding-bottom:17px;
}

QScrollBar::handle:vertical
{
    width:17px;
    background:rgba(0,0,0,25%);
    border-radius:4px;
    min-height:20;
}

QScrollBar::handle:vertical:hover
{
    width:17px;
    background:rgba(0,0,0,50%);
    border-radius:4px;
    min-height:20;
}

QScrollBar::add-line:vertical
{
    height:17px;width:17px;
    border-image:url(:/icon/res/scroll_down_normal.png);
    subcontrol-position:bottom;
}

QScrollBar::sub-line:vertical
{
    height:17px;width:17px;
    border-image:url(:/icon/res/scroll_up_normal.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
	height:17px;width:17px;
    border-image:url(:/icon/res/scroll_down_click.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
	height:17px;width:17px;
    border-image:url(:/icon/res/scroll_up_click.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}

QTreeView::item
{
	margin:3px;
}



QDateTimeEdit::down-arrow
{
	image: url(:/png/date_sel.png);
}

QDateTimeEdit::down-arrow:hover
{
	image: url(:/png/date_sel.png);
}

QDateTimeEdit::drop-down{
	border:none;
    background:transparent;
	padding-right:2px;
	width:30px;
	height:30px;
}
</string>
  </property>
  <widget class="QTableWidget" name="ip_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>IP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>端口</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>厂家</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>287</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
  	font-size:18px;
	color: rgb(255, 255, 255);
	border-color: rgba(255, 255, 255, 0);
	background-color: rgba(255, 255, 255, 0);
	font-family:Microsoft Yahei;
}</string>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPasswords">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>331</y>
     <width>198</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAccount">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>274</y>
     <width>198</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
  	font-size:18px;
	color: rgb(255, 255, 255);
	border-color: rgba(255, 255, 255, 0);
	background-color: rgba(255, 255, 255, 0);
	font-family:Microsoft Yahei;
}</string>
   </property>
   <property name="text">
    <string>密    码:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>680</y>
     <width>210</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	/*border-image: url(:/png/login.png);*/
	background-color: rgb(9, 163, 220);
    border-width: 0px;
border-radius:5px;
	color: white;
	font-size: 22px;
}

QPushButton:hover
{
	background-color: rgb(63, 197, 248);
/*	border-image: url(:/png/login_focus.png);*/
}

QPushButton:pressed
{
background-color:  rgb(63, 197, 248);
/*	border-image: url(:/png/login_focus.png);*/	
}</string>
   </property>
   <property name="text">
    <string>登  录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOther">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>680</y>
     <width>210</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	/*border-image: url(:/png/login.png);*/
	background-color: rgb(9, 163, 220);
    border-width: 0px;
border-radius:5px;
	color: white;
	font-size: 22px;
}

QPushButton:hover
{
	background-color: rgb(63, 197, 248);
/*	border-image: url(:/png/login_focus.png);*/
}

QPushButton:pressed
{
background-color:  rgb(63, 197, 248);
/*	border-image: url(:/png/login_focus.png);*/	
}</string>
   </property>
   <property name="text">
    <string>其它设备发现</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
